soccer/config/sim.xml

```diff
@@ -358,38 +355,40 @@
    <min value="1"/>
   </Health>
  </VisionFilter>
+ <Replanner>
+  <goalPosChangeThreshold value="0.025"/>
+  <goalVelChangeThreshold value="0.025"/>
+  <partialReplanLeadTime value="0.1"/>
+ </Replanner>
+ <LineKickPlanner>
+  <approachSpeed value="0.25"/>
+ </LineKickPlanner>
  <Capture>
+  <searchStartDist value="0"/>
+  <searchEndDist value="7"/>
+  <searchIncDist value="0.2"/>
+  <maxBallSpeedDirect value="0.1"/>
+  <maxApproachAngle value="0.3"/>
+  <maxOutrunBallSpeed value="0.7"/>
```

Comment:
Some of these don't exist?

Reply:
Fixed up all of the configs to match staging and cleared out unused ones.